src/extensions/FlagCommentControllerExtension.php

```diff
@@ -1,22 +1,40 @@
 <?php
```

Comment:
seems like using Tabs instead of spaces. can you please change it to spaces. please change in all the php files

Reply:
yup updated